components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/FormattingUtils.java

```diff
@@ -0,0 +1,30 @@
+package com.gradle.enterprise.cli;
+
+import java.time.Duration;
+import java.util.Locale;
+
+final class FormattingUtils {
+
+    private FormattingUtils() {
+    }
+
+    static String formatDuration(Duration duration) {
```

Comment:
You might notice this method is slightly different than before. I discovered this method also did not properly handle negative durations. The unit tests I added helped expose that.

Reply:
Nice.